core/claim.go

```diff
@@ -53,6 +53,13 @@ func GetClaimTypeVersionFromData(d *merkletree.Data) (c ClaimType, v uint32) {
 	return c, v
 }
 
+// HashString takes the first 31 bytes of a hash applied to name.
```

Comment:
I would change the comment:
`...hash applied to a string` instead of `...hash applied to name`

Reply:
done :)